### PR TITLE
Generic analysis form

### PR DESCRIPTION
**Pull request by gmrukwa**
This review introduces simple wrapper for the JSON-schema form. It will accept two URLs: for schema and layout, and exhibit `onSubmit` event in which data collected by the form will be processed.

Additionally, a bugfix for tests' results paths has been introduced.

Awaits merging of #171.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[Add generic form for scheduling analysis](https://trello.com/c/Zb9h1tPq/414-add-generic-form-for-scheduling-analysis)

File: src/app/generic-form/generic-form-fetch.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ * generic-form-fetch.service.spec.ts
+ * Unit tests for service fetching form definition.
+ *
+   Copyright 2017 Grzegorz Mrukwa
```
Comment (KapRom) [inline src/app/generic-form/generic-form-fetch.service.spec.ts]: maybe 2018? (all .ts files)

File: package.json [hunk 2]
```diff
@@ -58,13 +58,15 @@
     "@angular/compiler": "^5.2.6",
     "@angular/compiler-cli": "^5.2.6",
     "@angular/core": "^5.2.6",
+    "@angular/flex-layout": "2.0.0-beta.10-4905443",
```
Comment (Sebaxp95) [inline package.json]: There is new version

Comment (gmrukwa) [reply package.json]: There is, but we've checked experimentally and this one was the newest working.

File: src/app/generic-form/generic-form-fetch.service.spec.ts (hunk 1 above)
Comment (gmrukwa) [reply src/app/generic-form/generic-form-fetch.service.spec.ts]: Good catch :D

**Pull request merged**